README.md

```diff
@@ -1,147 +1,94 @@
+# Machine Learning Benchmarks
 
-# Machine Learning Benchmarks <!-- omit in toc -->
+[Developer Guide](docs/README.md)
 
 [![Build Status](https://dev.azure.com/daal/scikit-learn_bench/_apis/build/status/IntelPython.scikit-learn_bench?branchName=master)](https://dev.azure.com/daal/scikit-learn_bench/_build/latest?definitionId=8&branchName=master)
 
-**Machine Learning Benchmarks** contains implementations of machine learning algorithms
-across data analytics frameworks.  Scikit-learn_bench can be extended to add new frameworks
-and algorithms. It currently supports the [scikit-learn](https://scikit-learn.org/),
-[DAAL4PY](https://intelpython.github.io/daal4py/), [cuML](https://github.com/rapidsai/cuml),
-and [XGBoost](https://github.com/dmlc/xgboost) frameworks for commonly used
-[machine learning algorithms](#supported-algorithms).
+**Scikit-learn_bench** is a benchmarking tool for libraries and frameworks implementing Scikit-learn-like APIs and other workloads.
 
-## Follow us on Medium <!-- omit in toc -->
+## 📜 Table of content
 
-We publish blogs on Medium, so [follow us](https://medium.com/intel-analytics-software/tagged/machine-learning) to learn tips and tricks for more efficient data analysis. Here are our latest blogs:
+- [How to create Python environment for benchmarking](#-how-to-create-python-environment-for-benchmarking)
+- [How to use scikit-learn_bench](#-how-to-use-scikit-learn_bench)
+   - [Benchmarks runner](#benchmarks-runner)
+   - [Report generator](#report-generator)
+   - [Scikit-learn_bench high-level workflow](#scikit-learn_bench-high-level-workflow)
+- [Benchmark types](#-benchmark-types)
+- [Documentation tree](#-documentation-tree)
 
-- [Save Time and Money with Intel Extension for Scikit-learn](https://medium.com/intel-analytics-software/save-time-and-money-with-intel-extension-for-scikit-learn-33627425ae4)
-- [Superior Machine Learning Performance on the Latest Intel Xeon Scalable Processors](https://medium.com/intel-analytics-software/superior-machine-learning-performance-on-the-latest-intel-xeon-scalable-processor-efdec279f5a3)
-- [Leverage Intel Optimizations in Scikit-Learn](https://medium.com/intel-analytics-software/leverage-intel-optimizations-in-scikit-learn-f562cb9d5544)
```

Comment:
Why list of publication is removed? It is possible to move the corresponding page.